worker/src/main.rs

```diff
@@ -124,7 +134,13 @@ fn main() {
 
         println!("Worker Config: {:?}", config);
         let skip_ra = smatches.is_present("skip-ra");
-        worker(config.clone(), &shard, skip_ra);
+        worker(
+            config.clone(),
+            &shard,
+            skip_ra,
+            node_api_factory.clone(),
+            tokio_handle_accessor.clone(),
+        );
```

Comment:
node api factory and tokio handle accessor are passed into this function (thus removing another global variable access)